pkg/integrations/v2/app_o11y_exporter/receiver/receiver.go

```diff
@@ -76,25 +77,12 @@ func (ar *AppReceiver) ReceiverHandler(logger *log.Logger) http.Handler {
 		errChan := make(chan error)
 
 		for _, exporter := range ar.exporters {
-			wg.Add(2)
+			wg.Add(1)
 			go func(exp exporters.AppReceiverExporter) {
 				defer wg.Done()
-				// Metrics exporters run asynchronously when the self scrapping
-				// collects data
-				if de, ok := exp.(exporters.AppMetricsExporter); ok {
-					if err = de.Process(p); err != nil {
-						errChan <- err
-					}
-				}
-			}(exporter)
-
-			go func(exp exporters.AppReceiverExporter) {
-				defer wg.Done()
-				// Data exporters, export in sync with the user request
-				if de, ok := exp.(exporters.AppDataExporter); ok {
-					if err = de.Export(p); err != nil {
-						errChan <- err
-					}
+				if err = exp.Export(p); err != nil {
+					level.Error(logger).Log("msg", "exporter error", "exporter", exp.Name(), "error", err.Error())
```

Comment:
I think this could be debug level instead of log

Reply:
It's `error` level. Imho this is correct, exporter failures should be a unexpected and actionable issue, not ok to swallow them or hide under debug level